dataCompareR/_pkgdown.yml

```diff
@@ -0,0 +1,7 @@
+destination: docs
+
+reference:
+- contents:
```

Comment:
Could we add: 
```
- matches("print.dataCompareRobject")
- matches("summary.dataCompareRobject")
- matches("print.summary.dataCompareRobject")
```

Reply:
It doesn't seem to want to put them in the direct reference bit :/

They do exist though: https://krishanbhasin.github.io/dataCompareR/reference/summary.dataCompareRobject.html

Reply:
Nope, I jumped the gun, its there now!

![image](https://user-images.githubusercontent.com/8904718/79984900-b8029700-84a1-11ea-85e7-afe1dd8c2694.png)